atomic_defi_design/qml/Exchange/Trade/TradeBox/OrderForm.qml

```diff
@@ -230,6 +233,18 @@ FloatingBackground {
 
                 text: qsTr("Use custom minimum trade amount")
                 font.pixelSize: 13
+                onPressed: {
+                    if(!checked) { 
+                        _volumeRange.oldFirstValue =  _volumeRange.defaultFirstValue
+                    } 
+                    else { 
+                        _volumeRange.defaultFirstValue = API.app.trading_pg.orderbook.current_min_taker_vol
+                        _volumeRange.first.value = API.app.trading_pg.orderbook.current_min_taker_vol
+                    }
+                }
+                onCheckedChanged: {
+                    
+                }
```

Comment:
What the purpose of this code ? Why do you push code that literally does nothing ?

Reply:
sorry.